rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -154,31 +184,42 @@ object RholangCLI {
         throw e
     }
 
-  private def writeHumanReadable(fileName: String)(sortedTerm: Par): Unit = {
+  private def writeHumanReadable(fileName: String)(sortedTerm: Par): Try[Unit] = {
     val compiledFileName = fileName.replaceAll(".rho$", "") + ".rhoc"
-    new java.io.PrintWriter(compiledFileName) {
-      write(sortedTerm.toProtoString)
-      close()
-    }
-    println(s"Compiled $fileName to $compiledFileName")
+    Try({
+      new java.io.PrintWriter(compiledFileName) {
+        write(sortedTerm.toProtoString)
+        close()
+      }
+      println(s"Compiled $fileName to $compiledFileName")
+    })
   }
 
-  private def writeBinary(fileName: String)(sortedTerm: Par): Unit = {
+  private def writeBinary(fileName: String)(sortedTerm: Par): Try[Unit] = {
     val binaryFileName = fileName.replaceAll(".rho$", "") + ".bin"
-    val output         = new BufferedOutputStream(new FileOutputStream(binaryFileName))
-    output.write(sortedTerm.toByteString.toByteArray)
-    output.close()
-    println(s"Compiled $fileName to $binaryFileName")
+    Try({
+      val output         = new BufferedOutputStream(new FileOutputStream(binaryFileName))
+      output.write(sortedTerm.toByteString.toByteArray)
+      output.close()
+      println(s"Compiled $fileName to $binaryFileName")
+    })
   }
 
-  def evaluatePar(runtime: Runtime[Task])(par: Par)(implicit scheduler: Scheduler): Unit = {
+  def evaluatePar(runtime: Runtime[Task], quiet: Boolean = false)(
+      par: Par
+  )(implicit scheduler: Scheduler): Try[Unit] = {
     val evaluatorTask =
       for {
-        _      <- Task.now(printNormalizedTerm(par))
+        _ <- Task.now(if (!quiet) {
+              printNormalizedTerm(par)
+            })
         result <- Interpreter[Task].evaluate(runtime, par)
       } yield result
 
     waitForSuccess(evaluatorTask.runToFuture)
```

Comment:
```suggestion
    Try(waitForSuccess(evaluatorTask.runToFuture)).map {
      if (!quiet) {
        printStorageContents(runtime.space.store)
      }
    }
```

as ugly as it is using map for that. We shouldn't be using Try in the first place, but that's a discussion for a different PR. Try is still better than we had.